src/PKSim.Core/Services/IImportObservedDataTask.cs

```diff
@@ -1,10 +1,14 @@
-﻿using PKSim.Core.Model;
+﻿using OSPSuite.Core.Import;
+using PKSim.Core.Model;
 
 namespace PKSim.Core.Services
 {
    public interface IImportObservedDataTask
    {
       void AddObservedDataToProject();
       void AddObservedDataToProjectForCompound(Compound compound);
+      void AddObservedDataFromXmlToProjectForCompound(Compound subject);
```

Comment:
I don't understand the `FromXml` signature.  What use case is this?
Rename subject to compound